src/plugins/github/fetchGithubRepo.js

```diff
@@ -20,6 +20,57 @@ import {validateToken} from "./token";
 import {cacheIdForRepoId} from "./cacheId";
 import {type CacheProvider} from "../../backend/cache";
 
+type FetchRepoOptions = {|
+  +token: string,
```

Comment:
The token should be a validated GitHub token, not a string. It's the cleaner / more correct type signature.

I recognize this becomes stricter in that we'll now error if GitHub creates tokens that don't match our validator... but if that happens I'd rather just update the validation logic.

Reply:
Seems better as a separate PR to me. I'm keeping compatibility with the `fetchGithubRepo` arguments, which had an explicit reason to warn instead of error. Changing the API would break in a few places so this PR would complect.

My invocations of this function after the load refactor would be from `GithubTokens` already, casting back to strings.

Reply:
OK

Reply:
Tracking this issue here: https://github.com/sourcecred/sourcecred/issues/1626